interface/wx/toplevel.h

```diff
@@ -649,6 +649,10 @@ class wxTopLevelWindow : public wxNonOwnedWindow
         @param enable
             If @true (default) make the zoom button toggle full screen;
             if @false the button does only toggle zoom.
+        @param style
+            This parameter sets which elements will be hidden when the
+            user presses the full screen button. See ShowFullScreen()
+            for possible values.
```

Comment:
```suggestion
            for possible values. It is available since wxWidgets 3.1.6.
```

We probably should also mention in `wxFULLSCREEN_XXX` documentation which ones of them apply to `EnableFullScreenView()` too, and which ones are for `ShowFullScreen()` only.